src/plugins/intel_cpu/src/nodes/matmul.cpp

```diff
@@ -619,6 +624,12 @@ const std::vector<impl_desc_type>& MatMul::getPrimitivesPriority() {
     return implPriorities;
 }
 
+void MatMul::execute(dnnl::stream strm) {
+    if (prim) {
+        (*prim).execute(strm, primArgs);
+    }
+}
+
```

Comment:
Could you please elaborate on why `execute` method is implemented in this node and it is completely the same as the base class implementation? Is not it better just use the base implementation? Here and in the other similar cases in other nodes.

Reply:
Done